package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -92,8 +106,6 @@ class NucPairDist(AnalysisBase):
 
     Attributes
     ----------
-    times: numpy.ndarray
-        Simulation times for analysis.
```

Comment:
This should remain because it's created by AnalysisBase, isn't it?

Reply:
fixed in the recent versions